### PR TITLE
docs(prefix change): announce prefix change

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
 - Adds Announcements section.
 - Annoucements use our component and style - no MDX
 - Redid the landing page to use our components.
 - Excluded stories files from build

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1698

**How to test**  
1. Go to storybook
2. Check in Tegel Design System and Announcements

### CODE REVIEW

**Comment by alexandereneroth** [comment]
I feel like the announcement is tucked away and not easily discovered. 
What would you think about linking it on the start page in some way? 
Maybe even before the title with bold and with a 📢 or something. 🤔

**Comment by mJarsater** [comment]
> I feel like the announcement is tucked away and not easily discovered. What would you think about linking it on the start page in some way? Maybe even before the title with bold and with a 📢 or something. 🤔

Do you mean the prefix change or the tegel announcement? I think this is more in line with how we would want it in the future. The start page is just an introduction and then the announcements are under "Announcements".

**Comment by mJarsater** [comment]
> I feel like the announcement is tucked away and not easily discovered. What would you think about linking it on the start page in some way? Maybe even before the title with bold and with a 📢 or something. 🤔

Pushed an update now with a slimed down version of the announcement on the start page.

**Comment by alexandereneroth** [comment]
I meant just linking to it so it's discovered. Thought it was easily overlooked being tucked away in the announcements folder.
 
Like:

📣 [Were moving away from the sdds prefix, read more here!](https://www.google.com)
------------
# Tegel Design System

Bla bla bladida blablabla lorem ipsum dolor sit amet...

**Comment by mJarsater** [comment]
> I meant just linking to it so it's discovered. Thought it was easily overlooked being tucked away in the announcements folder.
> 
> Like:
> 
> ## 📣 [Were moving away from the sdds prefix, read more here!](https://www.google.com)
> # Tegel Design System
> Bla bla bladida blablabla lorem ipsum dolor sit amet...

Great idea! I like it

**Review by alexandereneroth — COMMENTED** [review]
I wonder why this one shows up now. 

Is it possible to remove it?

Screenshot 2023-05-31 at 14 44 11

File: core/src/stories/announcements/announce-tegel.stories.ts [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+                .mb-72 {
+                    margin-bottom: 72px;
+                }
+            </style>
+            <article class="tds-u-p2 tds-body-01">
+            <section>
+                        <p class="tds-body-01"><strong>Published: 2022-04-25</strong></p>
+                        <h1 class="tds-headline-02">Annoucing @scania</h1>
+                        <p>
+                            <b>TLDR;</b>The new @scania/tegel package is an improvement and continuation of the previous
```
Comment (timrombergjakobsson) [inline core/src/stories/announcements/announce-tegel.stories.ts:46]: Add space before you "The", or `<br />`after TLDR

File: core/src/stories/announcements/announce-tegel.stories.ts [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+                            @scania/component,
```
Comment (timrombergjakobsson) [inline core/src/stories/announcements/announce-tegel.stories.ts:47]:
```suggestion
                            @scania/components,
```

**Comment by mJarsater** [comment]
> I wonder why this one shows up now. 
> 
> 
> 
> Is it possible to remove it?
> 
> 
> 
> Screenshot 2023-05-31 at 14 44 11
> 
> 
> 
> 

Could you try it now?

**Comment by alexandereneroth** [comment]
Kind of want to make the other pages the same design now as well, what do you  think? Separate PR?

**Comment by mJarsater** [comment]
> Kind of want to make the other pages the same design now as well, what do you  think? Separate PR?
> 
> @mJarsater @timrombergjakobsson 

Yeah I had the same thought. I'd say separate PR. We don't want to block the prefix change epic by not merging this.

**Comment by timrombergjakobsson** [comment]
> > Kind of want to make the other pages the same design now as well, what do you  think? Separate PR?
> > @mJarsater @timrombergjakobsson
> 
> Yeah I had the same thought. I'd say separate PR. We don't want to block the prefix change epic by not merging this.

Sounds great!

File: core/src/stories/announcements/announce-tegel.stories.ts (hunk 1 above)
Comment (alexandereneroth) [inline core/src/stories/announcements/announce-tegel.stories.ts:47]:
```suggestion
                            <b>TLDR;</b> The new @scania/tegel package is an improvement and continuation of the previous
```

**Review by alexandereneroth — APPROVED** [review]

**Comment by alexandereneroth** [comment]
@mJarsater @timrombergjakobsson Approved from my end, but will give you a change to look before merge.

**Review by timrombergjakobsson — APPROVED** [review]
Looking 💯

**Review by timrombergjakobsson — APPROVED** [review]
Looking 💯

**Pull request merged**